src/ts/components/transaction/Overlay.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react'
+
+export const Overlay = () => {
+  return (
+    <svg xmlns="http://www.w3.org/2000/svg" width="246" height="2" viewBox="0 0 246 2">
```

Comment:
should we just keep this one as static e.g. xx.svg? or there is any dynamic stuff here?